doc/user/OPTIONS.md

```diff
@@ -13,7 +13,9 @@ Options|Default|Description
 \-\-no-background||Use with \-\-output to output a png file with a transparent background.
 -h, \-\-help||Print *help* and exit. Ignore `--verbose`.
 \-\-version||Show *version* information and exit. Ignore `--verbose`.
-\-\-readers-list||List available *readers* and exit. Ignore `--verbose`.
+\-\-list-readers||List available *readers* and exit. Ignore `--verbose`.
+\-\-list-bindings||List available *bindings* and exit. Ignore `--verbose`.
```

Comment:
not sure that it ignore `--verbose` though

Reply:
Indeed. It's copied from other list commands.

Reply:
Just checked, it's correct. The verbosity level is not taken into account yet